lib/src/navigation/maps_page.dart

```diff
@@ -1,87 +1,116 @@
-import 'package:dawnn_client/main.dart';
 import 'package:dawnn_client/src/network/objects/image.dart' as img;
-import 'package:dawnn_client/src/util/client_util.dart';
 import 'package:dawnn_client/src/util/network_util.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:location/location.dart';
 
+import 'image_viewer_page.dart';
+
 class MapPage extends StatefulWidget {
   @override
   State<StatefulWidget> createState() => _MapPageState();
 }
 
 class _MapPageState extends State<MapPage> {
-  final LatLng _center = const LatLng(45.521563, -122.677433);
```

Comment:
Map should start at general user location, this is a minor detail but should be fixed.